types/common.ts

```diff
@@ -2,7 +2,27 @@
  * TODO: implement the type
```

Comment:
This TODO should be deleted, as you updated `Candle` interface.

Reply:
sure